smartsim/settings/lsfSettings.py

```diff
@@ -215,10 +215,8 @@ def make_mpmd(self, jsrun_settings: t.Optional[JsrunSettings] = None):
                 "Colocated models cannot be run as a mpmd workload"
             )
 
-        if len(self.mpmd) == 0:
-            self.mpmd.append(self)
-        if jsrun_settings:
-            self.mpmd.append(jsrun_settings)
+        if settings:
```

Comment:
I don't see any tests or code that call `make_mpmd` without the argument, but the docstring is alerting me to a potential oddity:

```
This method will activate job execution through an ERF file.

Optionally, this method adds an instance of ``JsrunSettings`` to
the list of settings to be launched in the same ERF file.
```

@MattToast Do you think this docstring is correct that an ERF file obviates the need for a settings parameter?

Reply:
Went over this with @ashao and now understand the specifics. I was able to adjust the calling code (Orchestrator._build_run_settings_lsf) to avoid the need to have a parameter-less `make_mpmd`

I also created a follow-up ticket to consider unification of make_mpmd since all 5 are nearly identical except for the `if container` check